### ISSUE TITLE
uniform naming scheme for output files

### ISSUE THREAD

**Issue by eparejatobes** [bg7/BG7#9]
we need to make sure that all output files follow a reasonable naming scheme. What I'm proposing:

``` bash
${projectname}_bg7_annotation.$extension
```

where 
- `projectname` stands for the project name, set by the user
- `extension` is the corresponding file extension, currently one of `gbk, embl, gff, tsv, xml`

**Comment by pablopareja** [comment]
I totally agree with this, and please stop calling the xml file "the file for the viewer" !! :)
